src/test/java/com/monitorjbl/xlsx/StreamingReaderTest.java

```diff
@@ -417,4 +417,18 @@ public void testEncryption() throws Exception {
       }
     }
   }
+  
+  @Test
+  public void testStringCellValue () throws Exception {
```

Comment:
The one this comment is on: src/test/java/com/monitorjbl/xlsx/StreamingReaderTest.java